patches/display-language.diff

```diff
@@ -222,7 +222,7 @@ Index: code-server/lib/vscode/src/vs/workbench/services/localizations/browser/lo
 @@ -0,0 +1,28 @@
 +/*---------------------------------------------------------------------------------------------
 + *  Copyright (c) Coder Technologies. All rights reserved.
-+ *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Good call.

> I was also thinking that this copyright should be changed to Microsoft instead of Coder since we will remove the patch once it lands upstream anyway it might not be worth the effort.

Eh...I think it's minor so I say not worth it for now since we'll be removing this soon 🤞🏼